scss/print.scss

```diff
@@ -23,7 +23,7 @@
     }
 
     a[href^="/"]::after {
-        content: " (https://pub.uni-bielefeld.de" attr(href) ") ";
+        content: " (" attr(href) ") ";
```

Comment:
I'm not sure this rule should remain. It will now print the following if the domain name is not in the href attribute:
![image](https://user-images.githubusercontent.com/599538/31896135-4a501226-b813-11e7-9877-06fa15068452.png)

Not sure if relative paths are helpful.